runtime/dali/src/lib.rs

```diff
@@ -798,24 +800,16 @@ impl pallet_staking_rewards::Config for Runtime {
 
 /// The calls we permit to be executed by extrinsics
 pub struct BaseCallFilter;
-
 impl Contains<Call> for BaseCallFilter {
 	fn contains(call: &Call) -> bool {
-		if call_filter::Pallet::<Runtime>::contains(call) {
-			return false
-		}
-		!matches!(call, Call::Tokens(_) | Call::Indices(_) | Call::Democracy(_) | Call::Treasury(_))
+		!(call_filter::Pallet::<Runtime>::contains(call) ||
+			matches!(call, Call::Tokens(_) | Call::Indices(_) | Call::Treasury(_)))
```

Comment:
Should we add `Assets` to this and filter tokens?

Reply:
i removed only democracy, other than that filter is same. it is clear why tokens are filtered out. assets should be not as i think. balances? may or may not be, anyway they sudo secured.